packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -434,7 +436,13 @@ const Dropdown = ({
                 data-testid={test_id}
                 value={value || 0}
             />
-            <div ref={wrapper_ref} className={containerClassName()}>
+            <div
+                ref={wrapper_ref}
+                className={containerClassName()}
+                onMouseOver={should_open_on_hover ? () => setIsListVisible(true) : undefined}
+                onFocus={undefined}
```

Comment:
why is onFocus undefined

Reply:
I tried to remove sonarCloud warning, but it wasn't a successful attempt :( SonarCloaud require onFocus if you are using onMouseOver. Anyway, may attempt doesn't help